index.bs

```diff
@@ -1354,6 +1355,22 @@ The <dfn attribute for="XRInputSource">gripSpace</dfn> attribute is an {{XRSpace
 
 The {{gripSpace}} MUST be <code>null</code> if the input source isn't inherently trackable such as for input sources with a {{targetRayMode}} of {{XRTargetRayMode/gaze}} or {{XRTargetRayMode/screen}}.
 
+The <dfn attribute for="XRInputSource">profiles</dfn> attribute is a [=/list=] of [=input profile name=]s indicating both the preffered visual representation and the mapping of the buttons and axes of the {{XRInputSource/gamepad}} attribute of the {{XRInputSource}}. Names are given in decending order of preference.
```

Comment:
> the mapping of the buttons and axes of the {{XRInputSource/gamepad}} attribute of the {{XRInputSource}}

This is very specific to motion controllers with buttons.  Can we adjust this text so that it can also reflect variances in hand tracking capabilities even if no motion controller is present?